### ISSUE TITLE
Add signature to app

### ISSUE THREAD

**Issue by mottech20** [nextcloud/bookmarks#1941]
Currently the Bookmarks app reports to have no signature.json shipped with it.

```
php occ integrity:check-app bookmarks
  - EXCEPTION:
    - class: OC\IntegrityCheck\Exceptions\InvalidSignatureException
    - message: Signature data not found.
```

I don't know how you would have to do it though.

**Comment by marcelklehr** [comment]
I believe app integrity check is deprecated

**Comment by mottech20** [comment]
True, but it shows the ✔Featured icon on the Apps list (or the Apps Store) either. So if it's an official app, it would be good to have a signature at least if it doesn't require much effort. A nice-to-have.

**Comment by marcelklehr** [comment]
> So if it's an official app

heh, it's an open question what official means. I'll leave this open, though.
